chain/neutrino.go

```diff
@@ -457,7 +456,7 @@ func (s *NeutrinoClient) NotifyReceived(addrs []ltcutil.Address) error {
 	s.lastFilteredBlockHeader = nil
 
 	// Rescan with just the specified addresses.
-/* commented out until ltcsuite/neutrino is updated
+	/* commented out until ltcsuite/neutrino is updated
```

Comment:
this section was commented out last time we updated, due to un-solved dependency issues. We should un-comment it out now, to make it match btcsuite.